src/badge/badge.js

```diff
@@ -9,16 +10,19 @@ const Badge = props => {
     component,
     value,
     children,
+    element,
     ...attributes
   } = props;
 
+  if (element) return element;
```

Comment:
this is because in the listItem you can pass custom element. It's for backwards compatibility

```
<ListItem
  ...
  badge={{ element: <MyCustomElement> }}
/>
```